src/modules/sync/sagas.ts

```diff
@@ -71,6 +73,7 @@ export function* syncSaga() {
   yield takeLatest(DEPLOY_TO_LAND_SUCCESS, handleDeployToLandSuccess)
   yield takeLatest(CLEAR_DEPLOYMENT_SUCCESS, handleClearDeploymentSuccess)
   yield takeLatest(MARK_DIRTY, handleMarkDirty)
+  yield takeLatest(EDIT_PROJECT_THUMBNAIL, handleSaveThumbnail)
```

Comment:
to keep naming consisten this handler should be `handleEditProjectThumbnail`

Reply:
and also, when I create a new project, the `POST .../media` goes out before the first `PUT .../manifest` returning a 404 because the project doesn't exist yet.

Should we instead hook this handler to the `SAVE_PROJECT_SUCCESS` rather than the `EDIT_PROJECT_THUMBNAIL` to make sure this doesn't happen?

Reply:
SAVE_PROJECT_SUCCESS won't cotain the latest thumbnail, as a matter of fact it would contain the remote preview.png url for the first save

I'll keep thinking about how to solve both cases

Reply:
PROVISION_SCENE triggers handleSceneChange
handleSceneChange triggers SAVE_SCREENSHOT
SAVE_SCREENSHOT triggers EDIT_PROJECT_THUMBNAIL
(thumbnail updated at this point)

PROVISION_SCENE triggers SAVE_PROJECT_REQUEST
(thumbnail not updated at this point)